cmd/update-main.go

```diff
@@ -274,8 +277,8 @@ func mainUpdate(ctx *cli.Context) {
 		os.Exit(-1)
 	}
 
-	if older != time.Duration(0) {
-		log.Println(colorizeUpdateMessage(downloadURL, older))
+	if updateMsg := computeUpdateMessage(downloadURL, older); updateMsg != "" {
```

Comment:
`downloadURL` can be empty -> whenever it is the latest release or a dev-build (with timestamp greater than most recent release).